pkg/command/map_string_test.go

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of Cilium
+
+//go:build !privileged_tests
+// +build !privileged_tests
+
+package command
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetStringMapString(t *testing.T) {
```

Comment:
some more diversity in input and output values would probably be welcome here, for instance `k1=v2, k1=v1`. What's the result there? I'd argue we should maybe error out

Reply:
good point :100:, I actually miss this scenario in first implementation, thanks :100: